gnc/sonar/include/BVWrapper.h

```diff
@@ -0,0 +1,48 @@
+#include <iostream>
+#include <iterator>
+
+#include <bvt_sdk.h>
+
+#include <opencv2/core/core.hpp> 
+#include <opencv2/highgui/highgui.hpp>
+#include <opencv2/imgproc/imgproc.hpp>
+#include <ros/package.h>
+
+namespace bv{
+	const int BW8 = 0;
+	const int BW16 = 1;
+	const int COLOR = 2;
+	const int DEVICE = 3;
+	const int FILE = 4;
+	class BVWrapper{
+		private:
+
+			bool mapColors(BVTMagImage img, BVTColorImage& cimg);
+			bool getCVImages(BVTMagImage img, cv::Mat& image8, cv::Mat& image16, cv::Mat& image84);
+			void initializeSonar();
+
+			int dataSource = 0;
+			BVTHead head = NULL;
```

Comment:
One should use `nullptr` instead of `NULL`

It would be further wise to wrap each in a `std::shared_ptr<>`